.github/workflows/continuous-integration.yml

```diff
@@ -41,4 +41,7 @@ jobs:
 
       - name: Bazel Test All (opt)
         run: |
-          bazel test -c opt --noshow_progress --test_output=errors -- //xls/...
+          # Explicitly build the whole XLS excluding rules tagged as 'manual'
+          bazel build -c opt --noshow_progress --test_output=errors -- //xls/...
+          # Test everything except ZSTD module, this is done in a separate workflow
```

Comment:
I think we don't need that change anymore (i.e: bazel test should already build and test everything but manual zstd rules)

Reply:
Yes but we didn't mark the test rules as `manual`. It was done only for the following rules used for benchmarks:
* `xls_dslx_verilog`
* `xls_benchmark_ir`
* `verilog_library`
* `synthesize_rtl`
* `benchmark_synth`
* `place_and_route`

If we want to run the tests for the ZSTD module (rules `xls_dslx_test` and `cc_test`) then reverting this change back to the original is OK. However, I think we don't want to run those in the main CI workflow as per https://github.com/google/xls/pull/1315#issuecomment-2117654785.

In such case we will proceed with tagging all test rules (`xls_dslx_test` and `cc_test`) with the `manual` tag and that should allow us to go back to the original form of the main CI workflow. This will effectively mean that the only rules from the `//xls/modules/zstd` package that will be executed during `bazel test -c opt --noshow_progress --test_output=errors -- //xls/...` will be the `xls_dslx_library` build rules.